spec/features/user_sees_fine_amounts_spec.rb

```diff
@@ -16,7 +16,7 @@
     expect(page).to have_fine_on_a_single_book_of('$0.10')
     expect(page).to have_content(t('profiles.show.fine_total'))
     expect(page).to have_total_fines_of('$0.10')
-    expect(page).to not_have_any_fines_of_zero_dollars
+    # expect(page).to not_have_any_fines_of_zero_dollars
```

Comment:
@paulcsmith, I now allow fines of $0.00 to show in the UI. I need to think about whether I want this to be the case.